### ISSUE TITLE
pred_json for test dataset in TuSimple

### ISSUE THREAD

**Issue by TGLTommy** [fabvio/TuSimple-lane-classes#1]
@fabvio 

Hi, Fabvio, I got a problem which is how to generate a pred json file on TuSimple Test dataset ? 
I have run the LaneNet on TuSimple Training dataset, however I want to calculate the accuracy , FP, FN. Would you mind offering some suggestions ?  It really bothered me for several days. Thanks a lot.

**Comment by fabvio** [comment]
Hi, that's not the best place to ask since this is just an plugin to the dataset. AFAIK, you just need to create .json files equal to the ones in the training set, and process them with the provided script in tusimple repo. Try asking for more information in the official repo :)
